packages/url_launcher/url_launcher_web/lib/url_launcher_web.dart

```diff
@@ -24,14 +24,20 @@ String? _getUrlScheme(String url) => Uri.tryParse(url)?.scheme;
 bool _isSafariTargetTopScheme(String url) =>
     _safariTargetTopSchemes.contains(_getUrlScheme(url));
 
+
+bool _isSafari(navigator) =>
```

Comment:
Calling this method `_isSafari` is confusing the `UrlLauncherPlugin` constructor (because there's a private property with the same name), please find a better name for this method.

Also, remember to type the `navigator` parameter as `html.Navigator navigator`.

Reply:
Renaming the function should fix the following analyzer complaints:

```
  error - url_launcher_web.dart:40:17 - A value of type 'dynamic' can't be assigned to a variable of type 'bool'. Try changing the type of the variable, or casting the right-hand type to 'bool'. - invalid_assignment
  error - url_launcher_web.dart:40:17 - The expression doesn't evaluate to a function, so it can't be invoked. - invocation_of_non_function_expression
...
   info - url_launcher_web.dart:28:6 - The declaration '_isSafari' isn't referenced. Try removing the declaration of '_isSafari'. - unused_element
   info - url_launcher_web.dart:28:16 - Specify type annotations. - always_specify_types
```

Typing the `navigator` parameter should fix the following analysis problems:

```
  error - url_launcher_web.dart:29:5 - The operands of the operator '&&' must be assignable to 'bool'. - non_bool_operand
  error - url_launcher_web.dart:30:5 - The operands of the operator '&&' must be assignable to 'bool'. - non_bool_operand
  error - url_launcher_web.dart:31:6 - A negation operand must have a static type of 'bool'. Try changing the operand to the '!' operator. - non_bool_negation_expression
...
   info - url_launcher_web.dart:29:5 - Avoid method calls or property accesses on a "dynamic" target. - avoid_dynamic_calls
   info - url_launcher_web.dart:29:5 - Avoid method calls or property accesses on a "dynamic" target. - avoid_dynamic_calls
   info - url_launcher_web.dart:30:5 - Avoid method calls or property accesses on a "dynamic" target. - avoid_dynamic_calls
   info - url_launcher_web.dart:30:5 - Avoid method calls or property accesses on a "dynamic" target. - avoid_dynamic_calls
   info - url_launcher_web.dart:31:6 - Avoid method calls or property accesses on a "dynamic" target. - avoid_dynamic_calls
   info - url_launcher_web.dart:31:6 - Avoid method calls or property accesses on a "dynamic" target. - avoid_dynamic_calls
   info - url_launcher_web.dart:31:6 - Avoid method calls or property accesses on a "dynamic" target. - avoid_dynamic_calls
```